data/json/monsters/feral_humans.json

```diff
@@ -215,5 +215,135 @@
     "special_attacks": [ [ "TAZER", 10 ] ],
     "luminance": 500,
     "death_drops": "feral_security_death_drops_flashlight"
+  },
+  {
+    "id": "mon_feral_prepper",
+    "type": "MONSTER",
+    "name": { "str": "feral prepper" },
+    "description": "A quick glance of this person would show that they have prepared well for times like these.  Unfortunately, their bloodshot eyes, and the excluding treatment by the undead shows that they have turned feral.  One of their hands clasp the handle of their prepped improvised weapon, while the other clasps unto the grip of a handgun, treating both as clubs.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 75,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 5,
+    "dodge": 4,
+    "armor_bash": 7,
+    "armor_cut": 7,
+    "armor_bullet": 6,
+    "armor_acid": 4,
+    "vision_day": 50,
+    "vision_night": 3,
+    "harvest": "human",
+    "upgrades": { "into": "mon_feral_survivalist", "half_life": 60 },
+    "starting_ammo": { "9mm": 3 },
+    "special_attacks": [
+      [ "GRAB", 7 ],
+      {
+        "type": "gun",
+        "cooldown": 10,
+        "move_cost": 150,
+        "gun_type": "feral_mishandled_gun",
+        "ammo_type": "9mm",
+        "fake_skills": [ [ "gun", 1 ], [ "handgun", 1 ] ],
+        "fake_dex": 8,
```

Comment:
My biggest question is have you playtested this against a day one survivor.  How often does it hit and how many hits will  it take to kill an unarmored pc?

Reply:
I will experiment on this tomorrow, since I think I'm in a different timezone, and I'll need a different device-not this one.

Reply:
I made a saved character template and added an extra layer of emergency jacket and cargo pants with their standard survivor profession clothing, plus a baseball cap because why not. spawned the feral prepper and checked their damage output: It was too lethal for an early-mid enemy, so I will be replacing their combat skills from the mon-zom survivor with a mix of the feral axe and pipe human, because i think that's a more reasonable skill set. I will try again after this.

Reply:
I am half and mostly done with the playtesting. I have prepared the code, haven't committed it yet, but the feral prepper should have a melee lethality as same as the feral fireaxer, and the survivalist should have the same melee lethality as the feral 9mm guard. Still need to test the survivalist one more time because, I want to share with you, while testing the melee of the survivalist, they missed all their 4 shots, but the fifth one hit me. I didn't move so I stood there, and the funny thing is, they stopped dead in their tracks, and stared at me. They were atleast 4 or 5 tiles-unit distance away from my PC, and after a few seconds, they went from 'tracking', to fleeing. It was pretty funny and i wanted to share. Anyway, that means I should raise their morale from 50 to 75 so they still want to fight. That is all

Reply:
Alright, I have balanced their melee lethality, they should be similar to their counterparts, prepper for fire axe-er and survivalist for sec guard 9mm. The other device had 7 bullets for the survivalist, and they seem to miss 5 out of 7 bullets when shooting at total sight distance, and since I am not wanting to experiment on decreasing their dispersion right now, I am raising their bullet ammo from 5 to 6.